src/patternfly/components/Alert/alert.scss

```diff
@@ -49,6 +49,29 @@
   --pf-c-alert--m-info__icon--BackgroundColor: var(--pf-global--info-color--100);
   --pf-c-alert--m-info__title--Color: var(--pf-global--info-color--200);
 
+  // Inline modification
+  --pf-c-alert--m-inline__title--Color: var(--pf-global--Color--100);
+  --pf-c-alert--m-inline--BoxShadow: none;
+  --pf-c-alert--m-inline--BorderColor: var(--pf-global--BorderColor--300);
+  --pf-c-alert--m-inline--BorderLeftColor: var(--pf-global--info-color--200);
+  --pf-c-alert--m-inline--BorderStyle: solid;
+  --pf-c-alert--m-inline--BorderLeftStyle: solid;
+  --pf-c-alert--m-inline--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-alert--m-inline--BorderLeftWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-alert--m-inline__icon--BackgroundColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-alert__m-inline__icon--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-alert__m-inline__icon--PaddingRight: 0;
+  --pf-c-alert__m-inline__icon--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-alert__m-inline__icon--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-alert--m-success-inline__icon--Color: var(--pf-global--success-color--100);
```

Comment:
Can you separate these vars out into `--m-[state]--m-inline`, so not to confuse it as ineferring `.pf-m-[state]-inline`?